tendrl/commons/objects/cluster/atoms/stop_monitoring_services/__init__.py

```diff
@@ -0,0 +1,89 @@
+import etcd
+import time
+import uuid
+
+from tendrl.commons import objects
+from tendrl.commons.objects.job import Job
+from tendrl.commons.utils import log_utils as logger
+
+
+class StopMonitoringServices(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(StopMonitoringServices, self).__init__(*args, **kwargs)
+
+    def run(self):
+        integration_id = self.parameters['TendrlContext.integration_id']
+
+        try:
+            # Get the cluster nodes
+            nodes = NS._int.client.read("/clusters/%s/nodes" % integration_id)
+            child_job_ids = []
+            node_ids = []
+            for node in nodes.leaves:
+                node_id = node.key.split("/")[-1]
+                node_ids.append(node_id)
+                # Create jobs on nodes for stoping services
+                _job_id = str(uuid.uuid4())
+                params = {
+                    "Services[]": ["collectd"]
```

Comment:
This needs to be documented, stating that Tendrl will have full control over "collectd" service on the storage nodes, which includes shutting it down

@julienlim @mbukatov @nthomas-redhat  whats your take on this